src/track/track.cpp

```diff
@@ -744,6 +744,21 @@ void Track::setCuePoint(CuePosition cue) {
     emit cuesUpdated();
 }
 
+void Track::shiftCuePositions(double milliseconds) {
```

Comment:
I think we should hold the Track object free from this business logic.

Reply:
What do you mean?

Reply:
Never mind, I see that we have all the other 1 to n cue functions in Track. So it is OK.